gsa/src/web/pages/login/loginpage.js

```diff
@@ -121,33 +119,44 @@ const isIE11 = () =>
     : false;
 
 const LoginPage = props => {
```

Comment:
Maybe use `const LoginPage = () => {...` since there are no props?

Reply:
oh you are right